js/mexc3.js

```diff
@@ -3638,186 +3647,171 @@ module.exports = class mexc3 extends Exchange {
-        //         "confirmations":19,
-        //         "address":"0x55cbd73db24eafcca97369e3f2db74b2490586e6",
-        //         "state":"SUCCESS",
-        //         "tx_id":"0xc65a9b09e1b71def81bf8bb3ec724c0c1b2b4c82200c8c142e4ea4c1469fd789:0",
-        //         "require_confirmations":12,
-        //         "create_time":"2021-10-11T18:58:25.000+00:00",
-        //         "update_time":"2021-10-11T19:01:06.000+00:00"
-        //     }
+        // {
+        //     amount: '10',
+        //     coin: 'USDC-TRX',
+        //     network: 'TRX',
+        //     status: '5',
+        //     address: 'TSMcEDDvkqY9dz8RkFnrS86U59GwEZjfvh',
+        //     addressTag: null,
+        //     txId: '51a8f49e6f03f2c056e71fe3291aa65e1032880be855b65cecd0595a1b8af95b',
+        //     insertTime: '1664805021000',
+        //     unlockConfirm: '200',
+        //     confirmTimes: '203'
+        // }
         //
         // fetchWithdrawals
         //
-        //     {
-        //         "id":"4b450616042a48c99dd45cacb4b092a7",
-        //         "currency":"USDT-TRX",
-        //         "address":"TRHKnx74Gb8UVcpDCMwzZVe4NqXfkdtPak",
-        //         "amount":30.0,
-        //         "fee":1.0,
-        //         "remark":"this is my first withdrawal remark",
-        //         "state":"WAIT",
-        //         "create_time":"2021-10-11T20:45:08.000+00:00"
-        //     }
+        // {
+        //     id: 'adcd1c8322154de691b815eedcd10c42',
+        //     txId: '0xc8c918cd69b2246db493ef6225a72ffdc664f15b08da3e25c6879b271d05e9d0',
+        //     coin: 'USDC-MATIC',
+        //     network: 'MATIC',
+        //     address: '0xeE6C7a415995312ED52c53a0f8f03e165e0A5D62',
+        //     amount: '2',
+        //     transferType: '0',
+        //     status: '7',
+        //     transactionFee: '1',
+        //     confirmNo: null,
+        //     applyTime: '1664882739000',
+        //     remark: ''
+        //   }
         //
         // withdraw
         //
         //     {
         //         "withdrawId":"25fb2831fb6d4fc7aa4094612a26c81d"
         //     }
         //
-        const id = this.safeString2 (transaction, 'id', 'withdrawId');
+        const id = this.safeString (transaction, 'id');
         const type = (id === undefined) ? 'deposit' : 'withdrawal';
-        const timestamp = this.parse8601 (this.safeString (transaction, 'create_time'));
-        const updated = this.parse8601 (this.safeString (transaction, 'update_time'));
-        let currencyId = this.safeString (transaction, 'currency');
-        let network = undefined;
-        if ((currencyId !== undefined) && (currencyId.indexOf ('-') >= 0)) {
-            const parts = currencyId.split ('-');
-            currencyId = this.safeString (parts, 0);
-            const networkId = this.safeString (parts, 1);
-            network = this.safeNetwork (networkId);
-        }
+        const timestamp = this.safeNumber2 (transaction, 'insertTime', 'applyTime');
```

Comment:
@AlgoQ I'm going to fix it, but next time avoid parsing timestamps as floats and use `safeInteger` instead